pkg/policyengine/loadBalancer.go

```diff
@@ -193,7 +199,8 @@ func (lB *LoadBalancer) LookupWith(serviceSrc, serviceDst string, peers []string
 	policy := lB.getScheme(serviceSrc, serviceDst)
 
 	lB.updateState(serviceSrc, serviceDst)
-	plog.Infof("LoadBalancer lookup for serviceSrc %s serviceDst %s with policy %s with %+v", serviceSrc, serviceDst, policy, peers)
+	plog.Infof(
```

Comment:
align with usage in earlier `plog.Infof()` call; suggest curling up the message and bunching parameters into as few lines as possible.